dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -0,0 +1,53 @@
+export const menuOptions = [
+  {
+    group: { key: "dashboard", title: "Dashboard" },
+    submenu: [
+      {
+        name: "Overview",
+        submenu: true,
+        key: "overview",
+        submenuOf: "dashboard",
+        link: "overview",
+      },
+      {
+        name: "Analysis",
+        submenu: true,
+        key: "analysis",
+        submenuOf: "dashboard",
+        link: "analysis",
+      },
+      {
+        name: "All runs",
+        submenu: true,
+        key: "all_runs",
+        submenuOf: "dashboard",
+        link: "all_runs",
+      },
+      {
+        name: "Results",
+        submenu: true,
+        key: "results",
+        submenuOf: "dashboard",
+        link: "results",
+      },
+    ],
+  },
+  {
+    group: { key: "tools", title: "Tools" },
+    submenu: [
+      {
+        name: "Search",
+        submenu: true,
+        key: "search",
+        submenuOf: "tools",
+        link: "search",
+      },
+    ],
+  },
+];
+
+export const menuOptionsNonLoggedIn = [
+  { key: "dashboard", link: "dashboard", name: "Dashboard" },
+  { key: "search", link: "search", name: "Search" },
+  { key: "expore", link: "explore", name: "Explore" },
```

Comment:
It's not entirely clear how this format with the single `NavList`/`MenuItem` reference differs from the "logged-in" `menuOptions` above with `Nav`/`NavGroup`/`MenuItem`; but why does only "search" have the same link value? What do the `dashboard` and `explore` links do?

Reply:
For non-logged users, the menu options are different and they are not grouped so there is `NavList/MenuItem`.